src/tests/ftest/nvme/health.py

```diff
@@ -47,8 +48,18 @@ def test_monitor_for_large_pools(self):
         potential_num_pools = int((nvme_per_engine / (min_nvme_per_target * targets_per_engine)))
         actual_num_pools = min(max_num_pools, potential_num_pools)
 
+        # consider 1GiB RDB memory consume for MD-on-SSD
+        rdb_size = 1073741824
+        if self.server_managers[0].manager.job.using_control_metadata:
+            min_scm_per_pool = 104857600
```

Comment:
So RDB is always 1 GiB and min scm is always 100 MiB? Is there any risk of this changing?

Reply:
RDB default size is 1G and could be changed in the server yaml file by env, or it might be changed in the future development process, Min SCM size is similar, i just leave a value here for now.